spring-cloud-gcp-samples/spring-cloud-gcp-vision-ocr-demo/src/main/resources/templates/viewDocument.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE HTML>
+<html xmlns:th="https://www.thymeleaf.org">
+<head>
+  <title>View Document Text</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+  <link type="text/css" href="css/style.css" rel="stylesheet"/>
+</head>
+
+<body>
+<h1>Text from [[${gcsDocumentUrl}]]</h1>
+
+<p>
+  <a href="/status">Back to Status Page</a>
```

Comment:
Maybe you can make this page auto-refresh every 30 seconds or so, or tell the user to refresh manually.

Reply:
Done, added a note for user to refresh manually.